README.md

```diff
@@ -1,32 +1,48 @@
-A set of utilities to run the processCcd task on some CFHT data
+A set of utilities to run the processCcd task on some 
+CFHT data and DECam data
 and validate the astrometry of the results.
 
 Pre-requisites: install and declare the following
 1. pipe_tasks from the LSST DM stack (note that pipe_tasks is included with lsst_apps, which is the usual thing to install)
 2. obs_cfht from https://github.com/lsst/obs_cfht (this package is not included with lsst_apps); declare this with tag "current"
-3. validation_data_cfht from https://github.com/wmwv/validation_data_cfht
+3. validation_data_cfht from https://github.com/lsst/validation_data_cfht
+4. validation_data_decam from https://github.com/lsst/validation_data_decam
 
```

Comment:
Yes, thank you.  I rewrote this section a bit now that `obs_cfht` and `obs_decam` are in `lsstsw/etc/repos.yaml`.